cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -78,11 +78,14 @@ public void testRecentSnapshot() throws NotEnoughValidWindowsException {
     for (int i = 0; i < NUM_WINDOWS; i++) {
       assertEquals((NUM_WINDOWS - i) * WINDOW_MS, result.valuesAndExtrapolations().get(PE).window(i));
       for (Resource resource : Resource.values()) {
-        double expectedValue = resource == Resource.DISK ?
+        Collection<Integer> metricIds = KafkaMetricDef.resourceToMetricIds(resource);
```

Comment:
In this class, we occasionally use the word `snapshot` to refer to `windows` or `valuesAndExtrapolations`.